assets/queries/openAPI/link_object_with_both_operation_id_and_operation_ref/query.rego

```diff
@@ -23,6 +23,7 @@ CxPolicy[result] {
 	openapi_lib.check_openapi(doc) != "undefined"
 
 	link := doc.components.responses[r].links[l]
+	openapi_lib.content_allowed("", r)
```

Comment:
I think this does not makes sense, since components does not need response code, as you can see [here](https://swagger.io/specification/#components-object)